site/docs/v1/tech/reference/configuration.adoc

```diff
@@ -238,7 +238,7 @@ Deprecated names:
 * `fusionauth-app.https-port`
 
 [field]#fusionauth-app.http.read-timeout# [type]#[Integer]# [default]#defaults to `20,000`# [since]#Available since 1.37.0#::
-The HTTP read timeout in milliseconds that the server will wait to read data from
+The HTTP read timeout in milliseconds that the server will wait to read data from an incoming request.
```

Comment:
Good catch. I must have been mid-sentence and then started on something else. Oops.